egs/zeroth_korean/s5/run.sh

```diff
@@ -0,0 +1,265 @@
+    fi
+  fi
+fi
+
+
+if [ $stage -le 9 ]; then
+  echo "#### Triphone Training, LDA+MLLT+SAT ###########"
+  # Align the entire train_clean using the tri2 model
+  steps/align_si.sh  --nj $nj --cmd "$train_cmd" --use-graphs true \
+    data/train_clean data/lang_nosp exp/tri2 exp/tri2_ali_train_clean
+  
+  # Train tri3, which is LDA+MLLT+SAT on the entire train_clean
+  steps/train_sat.sh --cmd "$train_cmd" 4200 40000 \
+    data/train_clean data/lang_nosp exp/tri2_ali_train_clean exp/tri3
+  if $decode; then
+    utils/mkgraph.sh data/lang_nosp_test_tgsmall exp/tri3 exp/tri3/graph_nosp_tgsmall
+    nspk=$(wc -l <data/${test_set}/spk2utt)
+    steps/decode_fmllr.sh --nj $nspk --cmd "$decode_cmd" \
+      exp/tri3/graph_nosp_tgsmall data/${test_set} exp/tri3/decode_nosp_tgsmall_${test_set}
+    if $decode_rescoring; then
+      steps/lmrescore_const_arpa.sh \
+        --cmd "$decode_cmd" data/lang_nosp_test_{tgsmall,tglarge} \
+        data/$test_set exp/tri3/decode_nosp_{tgsmall,tglarge}_$test_set
+      steps/lmrescore_const_arpa.sh \
+        --cmd "$decode_cmd" data/lang_nosp_test_{tgsmall,fglarge} \
+        data/$test_set exp/tri3/decode_nosp_{tgsmall,fglarge}_$test_set
+    fi
+  fi
+fi 
+
+if [ $stage -le 10 ]; then
+  echo "#### Re-computing pronunciation model using tri3 model ###########"
+  # Now we compute the pronunciation and silence probabilities from training data,
+  # and re-create the lang directory.
+  # silence transition probability ...
+  steps/get_prons.sh --cmd "$train_cmd" \
+        data/train_clean data/lang_nosp exp/tri3
+  
+  utils/dict_dir_add_pronprobs.sh --max-normalize true \
+        data/local/dict_nosp \
+          exp/tri3/pron_counts_nowb.txt exp/tri3/sil_counts_nowb.txt \
+            exp/tri3/pron_bigram_counts_nowb.txt data/local/dict
+  
+  utils/prepare_lang.sh data/local/dict \
+        "<UNK>" data/local/lang_tmp data/lang
+  
+  local/format_lms.sh --src-dir data/lang data/local/lm
+  
+  utils/build_const_arpa_lm.sh \
+        data/local/lm/zeroth.lm.tg.arpa.gz data/lang data/lang_test_tglarge
+  utils/build_const_arpa_lm.sh \
+        data/local/lm/zeroth.lm.fg.arpa.gz data/lang data/lang_test_fglarge
+
+  if $decode; then
+    utils/mkgraph.sh data/lang_test_tgsmall exp/tri3 exp/tri3/graph_tgsmall
+    nspk=$(wc -l <data/${test_set}/spk2utt)
+    steps/decode_fmllr.sh --nj $nspk --cmd "$decode_cmd" \
+      exp/tri3/graph_tgsmall data/${test_set} exp/tri3/decode_tgsmall_${test_set}
+    if $decode_rescoring; then
+      steps/lmrescore_const_arpa.sh \
+        --cmd "$decode_cmd" data/lang_test_{tgsmall,tglarge} \
+        data/$test_set exp/tri3/decode_{tgsmall,tglarge}_$test_set
+      steps/lmrescore_const_arpa.sh \
+        --cmd "$decode_cmd" data/lang_test_{tgsmall,fglarge} \
+        data/$test_set exp/tri3/decode_{tgsmall,fglarge}_$test_set
+    fi
+  fi
+fi
+
+if [ $stage -le 11 ]; then
+
+  echo "#### SAT again on train_clean ###########"
```

Comment:
all echo statements should start with "$0:" so it's clear which script produced the output.

Reply:
now all echo statements start with "$0"